support-e2e/tests/tieredCheckout.test.ts

```diff
@@ -102,3 +108,49 @@ test.describe('Subscribe/Contribute via the Tiered checkout)', () => {
 		});
 	});
 });
+
+test.describe('Subscribe (S+) incl PromoCode via the Tiered checkout', () => {
```

Comment:
Could we just call the `SupporterPlus` as it is [in the API](https://product-catalog.guardianapis.com/product-catalog.json)?